components/Footer.vue

```diff
@@ -29,7 +29,7 @@
             <div class="flex flex-col flex-wrap mx-2">
                 <span>Contribute on </span>
                 <NuxtLink to="https://github.com/ourjapanlife/findadoc-web/" target="_blank" class="self-center">
-                    <img alt="github icon" class="github-icon w-12" src="@/assets/images/github_logo.png" />
+                    <img alt="github icon" class="github-icon w-12" src="@/assets/images/github-logo.png" />
```

Comment:
Can we use text here instead of an icon? It would be more lightweight. Plus the icon appears a bit fuzzy on screen.

Reply:
Can do! 
I actually added the GitHub logo as a placeholder until we have an updated design with it included.